mabl snippets/sendEnter.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Sends an enter key press to an element specified using a selector in
+ * the enterTargetSelector variable.
+ * 
+ * @param {object} - mablInputs Object containing input
+ *                   variables (mablInputs.variables.user)
+ * @param {function} callback - The callback function
+ */
+function mablJavaScriptStep(mablInputs, callback) {
+
+  let enterEvent = new KeyboardEvent("keydown", {
+    bubbles: true, cancelable: true, keyCode: 13
```

Comment:
is it worth adding a comment here with a link to a doc with keycodes so people can more easily update the script to suit their needs?

Reply:
Great idea. I was thinking about adding a link to https://keycode.info/